governance/votes/JanSky Executive Exercise June 12, 2023.md

```diff
@@ -0,0 +1,168 @@
+|-------------|-----------------------------------------------------------------------------------------------------------------------|-------|
+| 0xDefensor  | [0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9](https://etherscan.io/address/0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9) | 29.76 |
+| BONAPUBLICA | [0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3) | 29.76 |
+| QGov        | [0xB0524D8707F76c681901b782372EbeD2d4bA28a6](https://etherscan.io/address/0xB0524D8707F76c681901b782372EbeD2d4bA28a6) | 29.76 |
+| TRUE NAME   | [0x612f7924c367575a0edf21333d96b15f1b345a5d](https://etherscan.io/address/0x612f7924c367575a0edf21333d96b15f1b345a5d) | 29.76 |
+| UPMaker     | [0xbb819df169670dc71a16f58f55956fe642cc6bcd](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd) | 29.76 |
+| vigilant    | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) | 29.76 |
+| WBC         | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) | 20.16 |
+| PBG         | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) | 9.92  |
+| Bandhar     | [0xE83B6a503A94a5b764CCF00667689B3a522ABc21](https://etherscan.io/address/0xE83B6a503A94a5b764CCF00667689B3a522ABc21) | 7.68  |
+| Libertas    | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 7.04  |
+| PALC        | [0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A](https://etherscan.io/address/0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A) | 2.24  |
+| Harmony     | [0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2) | 1.92  |
+| VoteWizard  | [0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 1.6   |
+| Navigator   | [0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) | 0.32  |
+
+To learn more about Delegate Compensation, please read this [section](https://mips.makerdao.com/mips/details/MIP101#2-6-3-1-1-the-process-for-modifying-the-number-of-pd-and-rd-slots-must-generally-aim-to-have-changes-in-the-number-of-slots-happen-at-the-end-of-the-election-season-but-in-extraordinary-circumstances-which-must-be-specified-by-the-governance-scope-they-can-be-done-with-1-month-warning-) of the Atlas.
+
+### Andromeda Output Conduit
+
+As per this forum [post](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974/11) a new Output Conduit will be activated to allow Andromeda (RWA0151-A) to choose between Maker [PSMs](https://manual.makerdao.com/module-index/module-psm#trade-offs), if this executive proposal passes. 
```

Comment:
Could be made clearer. Consider someone that has no idea how or what Andromeda is. What are they doing that they can now choose between PSMs?

Reply:
Yeah, nail-coffin with the (RWA0151-A). Ldr made the exact same mistake, in the same form.